### PR TITLE
Update translation

### PR DESCRIPTION
**Pull request by jinhojang6**
- Update Chinese, Russian, Arabic, Korean, and Brazilian Portuguese translations
- Add Weibo to the footer as per the team's request

### CODE REVIEW

File: themes/navy/layout/partial/footer.ejs [hunk 1]
```diff
@@ -196,6 +196,20 @@
                         </svg>
                     </a>
                 </li>
+                <li class="ml-8 mt-8">
+                    <a class="hover:text-white" href="https://weibo.com/u/3928626105" target="_blank">
+                        <svg width="26" height="26" viewBox="0 0 26 26" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jakubgs) [inline themes/navy/layout/partial/footer.ejs:201]: Could we make this into a file rather than embedding it in the HTML?

Comment (jinhojang6) [reply themes/navy/layout/partial/footer.ejs:201]: All the svg logos(youtube, instagram, etc.) were embedded in the HTML. Let me make all of them into files if it is okay.

**Comment by jinhojang6** [comment]
@jakubgs Separated all the svg files in the footer

**Review by jakubgs — APPROVED** [review]
Sweet, thx.

**Pull request merged**